fire/cli/mtl.py

```diff
@@ -25,7 +29,41 @@
 from fire.cli import firedb
 
 # Typingelementer fra databaseAPIet.
-from fire.api.model import Punkt, Koordinat, PunktInformation, PunktInformationType
+from fire.api.model import (
+    GeometriObjekt,
+    Point,
+    Punkt,
+    Koordinat,
+    PunktInformation,
+    PunktInformationType,
+    Sag,
+    Sagsevent,
+    SagseventInfo,
+    Sagsinfo,
+)
+
+
+class Koordinatvrider:
```

Comment:
```suggestion
class KoordinatVrider:
```

Men jeg ville nok finde på et mere sigende navn, fx `WGS84Transformer`